add-to-core-party.lua

```diff
@@ -0,0 +1,65 @@
+local dialogs = require 'gui.dialogs'
+local utils = require 'utils'
+local viewscreen = dfhack.gui.getDFViewscreen(true)
+if viewscreen._type ~= df.viewscreen_dungeonmodest then
+    qerror("This script can only be used during adventure mode!")
+end
+
+local function addToCoreParty(nemesis)
+    -- Adds them to the party core members list
+    local party = df.global.adventure.interactions
+    -- problem: the "brain" icon deciding on manual/automatic control is somewhat broken.
+    -- research leads me to believe the data is stored per-unit or unit ID, need to figure out
+    -- where that data is stored exactly. Might be one of the unknown variables?
+    party.party_core_members:insert('#', nemesis.figure.id)
+    local extra_member_idx, _ = utils.linear_index(party.party_extra_members, nemesis.figure.id)
+    if extra_member_idx then
+        party.party_extra_members:erase(extra_member_idx)
+    end
+    -- Adds them to unretire list
+    nemesis.flags.ADVENTURER = true
+    nemesis.flags.ACTIVE_ADVENTURER = true
```

Comment:
is just adding them to the core party enough to turn `ACTIVE_ADVENTURER` to `true`? or is that just for the one that you're actively controlling?

Reply:
this is an oversight yah